src/pages/cody.tsx

```diff
@@ -22,7 +22,6 @@ import { HowCodyWorks } from '../components/cody/HowCodyWorks'
 import { ChooseYourLlmSection } from '../components/Enterprise/ChooseYourLlmSection'
 import { useAuthModal } from '../context/AuthModalContext'
 import { breakpoints } from '../data/breakpoints'
```

Comment:
Also lets fix the error on `line 1` that is causing the yarn lint fail:  "'useEffect' is defined but never used"